packages/sign-plugin/README.md

```diff
@@ -0,0 +1,62 @@
+# Grafana / Sign Plugin
+
+Sign Grafana plugins with ease.
+
+**ToC**
+
+- [Grafana / Sign Plugin](#grafana--sign-plugin)
+  - [Signing a plugin](#signing-a-plugin)
+    - [Sign a public plugin](#sign-a-public-plugin)
+    - [Sign a private plugin](#sign-a-private-plugin)
+      - [`npx`](#npx)
+      - [`yarn` (> 2.x)](#yarn--2x)
+  - [Contributing](#contributing)
+
+**Links**
+
+- [Signing plugins documentation](https://grafana.com/docs/grafana/latest/developers/plugins/sign-a-plugin/)
+
+**`@grafana/sign-plugin`** works on macOS, Windows and Linux.<br />
+If something doesn’t work, please [file an issue](https://github.com/grafana/create-plugin/issues/new).<br />
+If you have questions or need help, please ask in [GitHub Discussions](https://github.com/grafana/create-plugin/discussions).
+
+## Signing a plugin
+
+Signing a plugin allows Grafana to verify the authenticity of the plugin with signature verification. This gives users a way to make sure plugins haven’t been tampered with. All Grafana Labs-authored plugins, including Enterprise plugins, are signed.
+
+All plugins require a signature since Grafana 7.4.
+
+### Sign a public plugin
+
+In your plugin directory, sign the plugin with your Grafana API key. Grafana sign-plugin creates a MANIFEST.txt file in the dist directory of your plugin.
+
+```bash
+export GRAFANA_API_KEY=<YOUR_API_KEY>
+npx @grafana/sign-plugin
+```
+
+### Sign a private plugin
+
+In your plugin directory, run the following to create a MANIFEST.txt file in the dist directory of your plugin.
+
+```bash
+npx @grafana/sign-plugin --rootUrls https://example.com/grafana
```

Comment:
Not something necessarily for now: maybe it could be good to add some explanation on what the `--rootUrls` flag is exactly and how they should use it? (I don't think it's straightforward for everyone by the first sight.)

Reply:
Noted in #74 ! 🙏